packages/component-library/src/components/form/mt-switch/mt-switch.vue

```diff
@@ -10,6 +10,9 @@
             :name="formFieldName || identification"
             :checked="inputState"
             :disabled="isDisabled"
+            :aria-checked="inputState"
```

Comment:
Could you please write some tests to make sure your changes work? You can use the `toBeChecked` assertion of the [`jest-dom`](https://www.npmjs.com/package/@testing-library/jest-dom#tobechecked) package.